pkg/flexvolume/api_server.go

```diff
@@ -3,48 +3,58 @@ package flexvolume
 import (
 	"time"
 
-	"go.pedge.io/pb/go/google/protobuf"
-	"go.pedge.io/proto/rpclog"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/empty"
+	"go.pedge.io/dlog"
 	"golang.org/x/net/context"
 )
 
 type apiServer struct {
-	protorpclog.Logger
 	client Client
 }
 
 func newAPIServer(client Client) *apiServer {
-	return &apiServer{protorpclog.NewLogger("flexvolume.API"), client}
+	return &apiServer{client}
 }
 
-func (a *apiServer) Init(_ context.Context, _ *google_protobuf.Empty) (_ *google_protobuf.Empty, err error) {
-	defer func(start time.Time) { a.Log(nil, nil, err, time.Since(start)) }(time.Now())
+func (a *apiServer) Init(_ context.Context, _ *empty.Empty) (_ *empty.Empty, err error) {
```

Comment:
is this going to be backward compatible ?

Reply:
Yes this should be. It just changes the package from which Empty struct was being pulled from go.pedge.io/protobuf --> github.com/golang/protobuf